onepassword/client.py

```diff
@@ -444,10 +446,10 @@ def get_item(uuid: str | bytes, fields: str | bytes | list | None = None):
         elif isinstance(fields, str):
             item = {
                 fields: read_bash_return(
-                    "op item get {} --fields label={}".format(uuid, fields), single=False).rstrip('\n')
+                    "op item get {} --fields label={} {}".format(uuid, fields, vault_flag), single=False).rstrip('\n')
```

Comment:
Same as above plus what if it's empty?

```suggestion
                    "op item get {} --fields label={} --vault {}".format(uuid, fields, vault_flag), single=False).rstrip('\n')
```